packages/react-components/src/components/table-output-component.tsx

```diff
@@ -382,6 +382,10 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 this.timestampCol = timestampHeader.getColDef().field;
             }
         }
+
+        if (this.props.unitController.selectionRange) {
+            this.handleTimeSelectionChange(this.props.unitController.selectionRange);
```

Comment:
The selection of the index range is OK but ensuring the start index is visible does not always work.

I suspect it depends on timing, if fetching the selection index from the trace-server returns before or after the initial fetching of table lines.

I'm OK to fix this as a separate PR (or not fixed). The vertical scroll position of time graph rows and xy-chart checkbox tree is not restored either ;) The selected elements in xy-chart checkbox are also not restored. But we can expect some things to be reset when refreshing the page...